demo/tests/test_conditions.py

```diff
@@ -187,6 +193,15 @@ def test_readonly_field_not_displayed(self):
         self.assertTrue(form.is_valid(), form.errors)
         self.assertEqual(form.cleaned_data, {'checkbox': True})
 
+    def test_condition_field_doesnt_exist(self):
```

Comment:
Add description about the test:
* `wrong-conditions.json` viewed as an Employee (that's why, some fields are missing, b/c employee doesn't have the right to see them)
* `self.assertTrue()` is here to show we don't raise an exception anymore.